client/src/components/Flag.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export default function Flag() {
```

Comment:
Basically I think you could have this component take a prop of "visibilities" which it uses to decide which flags are hidden. That way when someone reads the flag component, it is clear that sometimes the flags are visible and sometimes they are not. The way it is written right now, someone, looking only at this component might assume that every flag piece is visible all the time.